globee/core.py

```diff
@@ -60,9 +60,9 @@ def check_required_fields(self):
             if not isinstance(total, (int, float)):
                 raise ValidationError('total is not an int, nor float!')
         except KeyError as e:
-            raise KeyError("%s not set" % e)
+            raise ValidationError("key '%s' not set" % e)
```

Comment:
Would be great if you could also change the tests like this one:
https://github.com/lovvskillz/django-globee/blob/master/globee/tests.py#L88